### DISCUSSION TITLE
apple virtualization framework

### DISCUSSION THREAD

**Discussion by mgrossu** [containers/podman#16626] [Ideas]
Hello, 

Since Apple introduced the new features in the Virtualization Framework in Ventura, I was thinking of replacing the podman backend from qemu with Apple's virtualization framework on macOS. So basically when using the command podman machine init, it will create a Linux VM but not with qemu, but with Apple's virtualization framework. 

Do you think it will be possible? 

I can take charge of it. 

My motivation is to use podman instead of Docker Desktop and at the same time, I do not want to use homebrew and qemu. Apple's virtualization framework is a higher-end API and it is possible to run Linux x86_64 binaries inside the Linux VM thanks to  Rosetta 2, so the containers can take also the advantage of it.

**Comment by Luap99** [comment]
This is planned, I am not sure what the current state is but there was already some initial work, https://github.com/containers/podman/pull/15199
cc @baude

**Comment by afbjorklund** [comment]
You can use podman with [lima](https://github.com/lima-vm/lima), which now has support for Virtualization.framework

It uses Fedora Cloud instead of Fedora CoreOS, but otherwise it's the same thing...
